tests/Makefile

```diff
@@ -169,6 +176,15 @@ $(TARGETS_WITH_TOOLCHAIN):
 build-tests-misc-with-builder-image:
 	$(MAKE) emulator-builder-exec CONTAINER_COMMAND="make -j\$$(nproc) build-tests-misc"
 
+create-machines $(CARTESI_CMIO_PATH): | $(CARTESI_IMAGES)
```

Comment:
If I type `make build-tests-all` twice, it will fail the second time here. I think doing a `rm -rf $(CARTESI_CMIO_PATH)` could fix this.

Reply:
Fixed `build-tests-all` dependency. It should build the templates only once now.